controls/pcidss_4_ocp4.yml

```diff
@@ -2035,12 +2035,26 @@ controls:
         on point-of-sale terminals).
       levels:
         - base
-      status: planned
+      status: automated
       notes: |-
-        The rules selected in this requirement are incomplete. Missing remediation and test
-        scenarios. They should include test scenarios and likely remediation or a warning
-        informing why a remediation is not present.
-      rules: []
+        Openshift should be configured to work with an external third-party identity provider.
+        Through the chosen identity provider, unique identifiers can be setup for each user.
+        See more at https://docs.openshift.com/container-platform/4.16/authentication/understanding-identity-provider.html
```

Comment:
We can make this link future proof by using `latest` instead of `4.16`.

https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html